client/dashboard/plugins/scheduled-updates/index.tsx

```diff
@@ -144,14 +144,17 @@ export const defaultView: View = {
 export default function PluginsScheduledUpdates() {
 	const [ view, setView ] = useState( defaultView );
 	const [ scheduleToDelete, setScheduleToDelete ] = useState< ScheduledUpdateRow | null >( null );
+	const [ isDeletingSchedule, setIsDeletingSchedule ] = useState( false );
 	const locale = useLocale();
 	const navigate = useNavigate();
 	const fields = useMemo( () => getFields( locale ), [ locale ] );
 
 	const { isLoading, scheduledUpdates } = useScheduledUpdates();
 	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
-	const { mutate: deleteSchedule, isPending: isDeletingSchedule } = useMutation(
-		hostingUpdateScheduleDeleteMutation()
+	const { mutateAsync: deleteSchedules } = useDeleteSchedules(
+		scheduleToDelete ? [ scheduleToDelete.site.ID ] : [],
+		scheduleToDelete?.scheduleId ?? '',
+		{ optimisticHosting: false }
```

Comment:
I disabled this for now. I just don't see the UI being any better with this enabled. In fact, I think it's worse - the user sees the list updated in the background but still waits for the operation to complete. It just doesn't feel like a good UX.

Happy to re-enable or strip out completely. I will go ahead and merge this PR now as I have another branched off this one. I will do a follow-up depending on feedback. cc @Automattic/lego if you can take a look at the description and the "with/without" screencasts pls 🙏